forge/forge/verify/compare.py

```diff
@@ -12,56 +12,69 @@
 from loguru import logger
 from scipy.spatial import distance
 
-from forge.verify.config import VerifyConfig
 from forge.tensor import narrow_forge_tensor_to_pytorch
 
 # Compares golden and calculated tensors. Using allclose for scalar values, rogerstanimoto for bool tensors, pcc otherwise
 def compare_with_golden(
     golden: Union[torch.Tensor, tf.Tensor, tf.Variable],
     calculated: torch.Tensor,
-    verify_cfg: VerifyConfig = VerifyConfig(),
+    pcc: float = 0.99,
+    rtol: float = 1e-05,
+    atol: float = 1e-08,
+    dissimilarity_threshold: float = 1e-03,
 ):
     if golden.dtype == torch.bool:
-        return compare_with_golden_bool(golden, calculated, verify_cfg)
+        return compare_with_golden_bool(golden, calculated, dissimilarity_threshold)
     if golden.flatten().size() != (1,):  # PCC for single values doesn't work
-        return compare_with_golden_pcc(golden, calculated, verify_cfg)
+        return compare_with_golden_pcc(golden, calculated, pcc)
     else:
         # For scalar values, we can't calculate PCC, but we can compare golden and calculated values using relative and absolute tolerances
         golden = golden.flatten()[0]
         calculated = calculated.flatten()[0]
-        return torch.allclose(
-            golden, calculated, rtol=verify_cfg.rtol[golden.dtype], atol=verify_cfg.atol[golden.dtype]
-        )
+
+        all_close = torch.allclose(golden, calculated, rtol=rtol, atol=atol)
+        if not all_close:
+            req_atol, req_rtol = compute_required_tolerances(golden, calculated)
+            logger.error("Tensor mismatch. Current rtol={}, atol={}", rtol, atol)
```

Comment:
Please change the messages, it looks confusing to me. I would formulate it something like:
```python
logger.error("Tensor mismatch. Required rtol={}, atol={}", rtol, atol)
logger.error("Observed maximum relative diff: {}, maximum absolute diff: {}", req_rtol, req_atol)
```